frontend/javascripts/oxalis/view/right-border-tabs/bounding_box_tab.js

```diff
@@ -1,30 +1,100 @@
-/**
- * tracing_settings_view.js
- * @flow
- */
-import { Tooltip } from "antd";
+// @flow
+import { Button, Form, Modal, Tooltip } from "antd";
 import { PlusSquareOutlined } from "@ant-design/icons";
 import { useSelector, useDispatch } from "react-redux";
 import React, { useState } from "react";
 import _ from "lodash";
 
-import { setPositionAction } from "oxalis/model/actions/flycam_actions";
-import type { Vector3, Vector6, BoundingBoxType } from "oxalis/constants";
+import { DatasetNameFormItem } from "admin/dataset/dataset_components";
 import { UserBoundingBoxInput } from "oxalis/view/components/setting_input_views";
-import { getSomeTracing } from "oxalis/model/accessors/tracing_accessor";
+import type { Vector3, Vector6, BoundingBoxType } from "oxalis/constants";
 import {
   changeUserBoundingBoxAction,
   addUserBoundingBoxAction,
   deleteUserBoundingBoxAction,
 } from "oxalis/model/actions/annotation_actions";
+import { getActiveSegmentationTracingLayer } from "oxalis/model/accessors/volumetracing_accessor";
+import { getBaseSegmentationName } from "oxalis/view/right-border-tabs/segments_tab/segments_view_helper";
+import { getSomeTracing } from "oxalis/model/accessors/tracing_accessor";
+import { setPositionAction } from "oxalis/model/actions/flycam_actions";
+import { startGlobalizeFloodfillsJob } from "admin/admin_rest_api";
+import ExportBoundingBoxModal from "oxalis/view/right-border-tabs/export_bounding_box_modal";
+import Toast from "libs/toast";
 import * as Utils from "libs/utils";
+import features from "features";
 
-import ExportBoundingBoxModal from "oxalis/view/right-border-tabs/export_bounding_box_modal";
+const GLOBALIZE_FLOODFILL_REGEX = /Limits of flood-fill \(source_id=(\d+), target_id=(\d+), seed=([\d,]+), timestamp=(\d+)\)/;
```

Comment:
Maybe put this regex next to the spot where the pattern is used to name bounding boxes, to avoid that the two desynchronize?

Reply:
It'll probably work best if the string interpolation in the volumetracing_saga.js is extracted into a function so that the regex can be put next to it.